packages/camera/camera_avfoundation/ios/Classes/FLTThreadSafeEventChannel.m

```diff
@@ -21,8 +21,9 @@ - (instancetype)initWithEventChannel:(FlutterEventChannel *)channel {
 
 - (void)setStreamHandler:(NSObject<FlutterStreamHandler> *)handler
               completion:(void (^)(void))completion {
+  __weak typeof(self) weakSelf = self;
   FLTEnsureToRunOnMainQueue(^{
-    [self.channel setStreamHandler:handler];
+    [weakSelf.channel setStreamHandler:handler];
     completion();
```

Comment:
Should the completion be run if this is dead?

Reply:
Me being lazy. Should've used `if (!strongSelf) return;` here.  

Looking at the current usage, running completion should be fine, but I think it's better not to run it.